components/youtube_analytics_api/actions/list-channel-reports/list-channel-reports.mjs

```
@@ -51,11 +54,13 @@
       }
 
       if (channelReportType === PLAYLIST_BASIC_STATS.value) {
-        const supportedFilters = PLAYLIST_BASIC_STATS.metadata.filters
-          .reduce((acc, filter) => ({
+        const supportedFilters = PLAYLIST_BASIC_STATS.metadata.filters.reduce(
+          (acc, filter) => ({
             ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)